### ISSUE TITLE
captainkube docker build fails at go build

### ISSUE THREAD

**Issue by schdief** [Azure/phippyandfriends#24]
tried the tutorial (https://cloudblogs.microsoft.com/opensource/2018/11/27/tutorial-azure-devops-setup-cicd-pipeline-kubernetes-docker-helm/) and the docker build of captainkube fails at step 5

error:
```
Step 5/10 : RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
 ---> Running in bf8883fd3257
# github.com/sabbour/phippy
./main.go:79:50: client.Core undefined (type *kubernetes.Clientset has no field or method Core)
The command '/bin/sh -c CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .' returned a non-zero code: 2
```

full output locally:
```
Steves-MBP:captainkube schdief$ docker build .
Sending build context to Docker daemon  22.53kB
Step 1/10 : FROM golang:1.10.3 as builder
1.10.3: Pulling from library/golang
55cbf04beb70: Pull complete 
1607093a898c: Pull complete 
9a8ea045c926: Pull complete 
d4eee24d4dac: Pull complete 
9c35c9787a2f: Pull complete 
6a66653f6388: Pull complete 
102f6b19f797: Pull complete 
Digest: sha256:3c54fa85d6262d2ef7695ee2f8793f1f4f9809ce4a08ca2e213235ef4cfdcb66
Status: Downloaded newer image for golang:1.10.3
 ---> d0e7a411e3da
Step 2/10 : WORKDIR /go/src/github.com/sabbour/phippy
 ---> Running in 90f7be4b8813
Removing intermediate container 90f7be4b8813
 ---> 8aa954f9ba20
Step 3/10 : COPY main.go .
 ---> 14b99e345573
Step 4/10 : RUN go get -d -v
 ---> Running in 053caf014328
Fetching https://k8s.io/api/core/v1?go-get=1
Parsing meta tags from https://k8s.io/api/core/v1?go-get=1 (status code 200)
get "k8s.io/api/core/v1": found meta tag get.metaImport{Prefix:"k8s.io/api", VCS:"git", RepoRoot:"https://github.com/kubernetes/api"} at https://k8s.io/api/core/v1?go-get=1
get "k8s.io/api/core/v1": verifying non-authoritative meta tag
Fetching https://k8s.io/api?go-get=1
Parsing meta tags from https://k8s.io/api?go-get=1 (status code 200)
k8s.io/api (download)
Fetching https://k8s.io/apimachinery/pkg/api/resource?go-get=1
Parsing meta tags from https://k8s.io/apimachinery/pkg/api/resource?go-get=1 (status code 200)
get "k8s.io/apimachinery/pkg/api/resource": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/api/resource?go-get=1
get "k8s.io/apimachinery/pkg/api/resource": verifying non-authoritative meta tag
Fetching https://k8s.io/apimachinery?go-get=1
Parsing meta tags from https://k8s.io/apimachinery?go-get=1 (status code 200)
k8s.io/apimachinery (download)
Fetching https://k8s.io/apimachinery/pkg/apis/meta/v1?go-get=1
Parsing meta tags from https://k8s.io/apimachinery/pkg/apis/meta/v1?go-get=1 (status code 200)
get "k8s.io/apimachinery/pkg/apis/meta/v1": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/apis/meta/v1?go-get=1
get "k8s.io/apimachinery/pkg/apis/meta/v1": verifying non-authoritative meta tag
github.com/google/gofuzz (download)
Fetching https://k8s.io/apimachinery/pkg/fields?go-get=1
Parsing meta tags from https://k8s.io/apimachinery/pkg/fields?go-get=1 (status code 200)
get "k8s.io/apimachinery/pkg/fields": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/fields?go-get=1
get "k8s.io/apimachinery/pkg/fields": verifying non-authoritative meta tag
Fetching https://k8s.io/klog?go-get=1
Parsing meta tags from https://k8s.io/klog?go-get=1 (status code 200)
get "k8s.io/klog": found meta tag get.metaImport{Prefix:"k8s.io/klog", VCS:"git", RepoRoot:"https://github.com/kubernetes/klog"} at https://k8s.io/klog?go-get=1
k8s.io/klog (download)
Fetching https://k8s.io/apimachinery/pkg/runtime?go-get=1
Parsing meta tags from https://k8s.io/apimachinery/pkg/runtime?go-get=1 (status code 200)
get "k8s.io/apimachinery/pkg/runtime": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/runtime?go-get=1
get "k8s.io/apimachinery/pkg/runtime": verifying non-authoritative meta tag
Fetching https://k8s.io/apimachinery/pkg/runtime/schema?go-get=1
Parsing meta tags from https://k8s.io/apimachinery/pkg/runtime/schema?go-get=1 (status code 200)
get "k8s.io/apimachinery/pkg/runtime/schema": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/runtime/schema?go-get=1
get "k8s.io/apimachinery/pkg/runtime/schema": verifying non-authoritative meta tag
Fetching https://k8s.io/apimachinery/pkg/types?go-get=1
Parsing meta tags from https://k8s.io/apimachinery/pkg/types?go-get=1 (status code 200)
get "k8s.io/apimachinery/pkg/types": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/types?go-get=1
get "k8s.io/apimachinery/pkg/types": verifying non-authoritative meta tag
Fetching https://k8s.io/apimachinery/pkg/util/intstr?go-get=1
Parsing meta tags from https://k8s.io/apimachinery/pkg/util/intstr?go-get=1 (status code 200)
get "k8s.io/apimachinery/pkg/util/intstr": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/util/intstr?go-get=1
get "k8s.io/apimachinery/pkg/util/intstr": verifying non-authoritative meta tag
Fetching https://k8s.io/client-go/kubernetes?go-get=1
Parsing meta tags from https://k8s.io/client-go/kubernetes?go-get=1 (status code 200)
get "k8s.io/client-go/kubernetes": found meta tag get.metaImport{Prefix:"k8s.io/client-go", VCS:"git", RepoRoot:"https://github.com/kubernetes/client-go"} at https://k8s.io/client-go/kubernetes?go-get=1
get "k8s.io/client-go/kubernetes": verifying non-authoritative meta tag
Fetching https://k8s.io/client-go?go-get=1
Parsing meta tags from https://k8s.io/client-go?go-get=1 (status code 200)
k8s.io/client-go (download)
Fetching https://k8s.io/client-go/rest?go-get=1
Parsing meta tags from https://k8s.io/client-go/rest?go-get=1 (status code 200)
get "k8s.io/client-go/rest": found meta tag get.metaImport{Prefix:"k8s.io/client-go", VCS:"git", RepoRoot:"https://github.com/kubernetes/client-go"} at https://k8s.io/client-go/rest?go-get=1
get "k8s.io/client-go/rest": verifying non-authoritative meta tag
Fetching https://k8s.io/utils/integer?go-get=1
Parsing meta tags from https://k8s.io/utils/integer?go-get=1 (status code 200)
get "k8s.io/utils/integer": found meta tag get.metaImport{Prefix:"k8s.io/utils", VCS:"git", RepoRoot:"https://github.com/kubernetes/utils"} at https://k8s.io/utils/integer?go-get=1
get "k8s.io/utils/integer": verifying non-authoritative meta tag
Fetching https://k8s.io/utils?go-get=1
Parsing meta tags from https://k8s.io/utils?go-get=1 (status code 200)
k8s.io/utils (download)
Fetching https://k8s.io/client-go/tools/cache?go-get=1
Parsing meta tags from https://k8s.io/client-go/tools/cache?go-get=1 (status code 200)
get "k8s.io/client-go/tools/cache": found meta tag get.metaImport{Prefix:"k8s.io/client-go", VCS:"git", RepoRoot:"https://github.com/kubernetes/client-go"} at https://k8s.io/client-go/tools/cache?go-get=1
get "k8s.io/client-go/tools/cache": verifying non-authoritative meta tag
Removing intermediate container 053caf014328
 ---> 124fc3efabc6
Step 5/10 : RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
 ---> Running in bf8883fd3257
# github.com/sabbour/phippy
./main.go:79:50: client.Core undefined (type *kubernetes.Clientset has no field or method Core)
The command '/bin/sh -c CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .' returned a non-zero code: 2
```

full output on azure devops (set up like shown in tutorial as mentioned above):
```
##[section]Starting: docker build
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.148.2
Author       : Microsoft Corporation
Help         : [More Information](https://go.microsoft.com/fwlink/?LinkID=613738)
==============================================================================
Generating script.
Script contents:
docker build -f /home/vsts/work/1/s/captainkube/Dockerfile -t phippyandfriendsschdieflaw.azurecr.io/captainkube:62 -t phippyandfriendsschdieflaw.azurecr.io/captainkube:latest /home/vsts/work/1/s/captainkube
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
(node:6644) Warning: Use Cipheriv for counter mode of aes-256-ctr
========================== Starting Command Output ===========================
[command]/bin/bash --noprofile --norc /home/vsts/work/_temp/2169ccf1-a3bc-49c0-b6cb-65885effdf3c.sh
Sending build context to Docker daemon  22.53kB

Step 1/10 : FROM golang:1.10.3 as builder
1.10.3: Pulling from library/golang
55cbf04beb70: Pulling fs layer
1607093a898c: Pulling fs layer
9a8ea045c926: Pulling fs layer
d4eee24d4dac: Pulling fs layer
9c35c9787a2f: Pulling fs layer
6a66653f6388: Pulling fs layer
102f6b19f797: Pulling fs layer
9c35c9787a2f: Waiting
d4eee24d4dac: Waiting
102f6b19f797: Waiting
6a66653f6388: Waiting
9a8ea045c926: Verifying Checksum
9a8ea045c926: Download complete
1607093a898c: Verifying Checksum
1607093a898c: Download complete
55cbf04beb70: Verifying Checksum
55cbf04beb70: Download complete
d4eee24d4dac: Download complete
9c35c9787a2f: Verifying Checksum
9c35c9787a2f: Download complete
102f6b19f797: Verifying Checksum
102f6b19f797: Download complete
6a66653f6388: Verifying Checksum
6a66653f6388: Download complete
55cbf04beb70: Pull complete
1607093a898c: Pull complete
9a8ea045c926: Pull complete
d4eee24d4dac: Pull complete
9c35c9787a2f: Pull complete
6a66653f6388: Pull complete
102f6b19f797: Pull complete
Digest: sha256:3c54fa85d6262d2ef7695ee2f8793f1f4f9809ce4a08ca2e213235ef4cfdcb66
Status: Downloaded newer image for golang:1.10.3
 ---> d0e7a411e3da
Step 2/10 : WORKDIR /go/src/github.com/sabbour/phippy
 ---> Running in 79adc6493dfe
Removing intermediate container 79adc6493dfe
 ---> 5d981ac7fb98
Step 3/10 : COPY main.go .
 ---> 1201b0f4fed7
Step 4/10 : RUN go get -d -v
 ---> Running in e388b0cd6310
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;api&#x2F;core&#x2F;v1?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;api&#x2F;core&#x2F;v1?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;api&#x2F;core&#x2F;v1&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;api&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;api&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;api&#x2F;core&#x2F;v1?go-get=1‌
get "k8s.io/api/core/v1": verifying non-authoritative meta tag
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;api?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;api?go-get=1 (status code 200)‌
k8s.io&#x2F;api (download)‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;api&#x2F;resource?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;api&#x2F;resource?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;api&#x2F;resource&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;apimachinery&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;apimachinery&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;api&#x2F;resource?go-get=1‌
get "k8s.io/apimachinery/pkg/api/resource": verifying non-authoritative meta tag
Fetching https://k8s.io/apimachinery?go-get=1
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery?go-get=1 (status code 200)‌
k8s.io&#x2F;apimachinery (download)‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;apis&#x2F;meta&#x2F;v1?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;apis&#x2F;meta&#x2F;v1?go-get=1 (status code 200)‌
get "k8s.io/apimachinery/pkg/apis/meta/v1": found meta tag get.metaImport{Prefix:"k8s.io/apimachinery", VCS:"git", RepoRoot:"https://github.com/kubernetes/apimachinery"} at https://k8s.io/apimachinery/pkg/apis/meta/v1?go-get=1
get "k8s.io/apimachinery/pkg/apis/meta/v1": verifying non-authoritative meta tag
github.com&#x2F;google&#x2F;gofuzz (download)‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;fields?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;fields?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;fields&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;apimachinery&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;apimachinery&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;fields?go-get=1‌
get "k8s.io/apimachinery/pkg/fields": verifying non-authoritative meta tag
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;klog?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;klog?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;klog&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;klog&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;klog&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;klog?go-get=1‌
k8s.io&#x2F;klog (download)‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;apimachinery&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;apimachinery&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime?go-get=1‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime&quot;: verifying non-authoritative meta tag‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime&#x2F;schema?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime&#x2F;schema?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime&#x2F;schema&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;apimachinery&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;apimachinery&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime&#x2F;schema?go-get=1‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;runtime&#x2F;schema&quot;: verifying non-authoritative meta tag‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;types?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;types?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;types&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;apimachinery&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;apimachinery&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;types?go-get=1‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;types&quot;: verifying non-authoritative meta tag‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;util&#x2F;intstr?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;util&#x2F;intstr?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;util&#x2F;intstr&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;apimachinery&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;apimachinery&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;util&#x2F;intstr?go-get=1‌
get &quot;k8s.io&#x2F;apimachinery&#x2F;pkg&#x2F;util&#x2F;intstr&quot;: verifying non-authoritative meta tag‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;kubernetes?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;kubernetes?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;client-go&#x2F;kubernetes&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;client-go&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;client-go&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;kubernetes?go-get=1‌
get "k8s.io/client-go/kubernetes": verifying non-authoritative meta tag
Fetching https://k8s.io/client-go?go-get=1
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;client-go?go-get=1 (status code 200)‌
k8s.io&#x2F;client-go (download)‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;rest?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;rest?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;client-go&#x2F;rest&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;client-go&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;client-go&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;rest?go-get=1‌
get &quot;k8s.io&#x2F;client-go&#x2F;rest&quot;: verifying non-authoritative meta tag‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;utils&#x2F;integer?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;utils&#x2F;integer?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;utils&#x2F;integer&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;utils&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;utils&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;utils&#x2F;integer?go-get=1‌
get &quot;k8s.io&#x2F;utils&#x2F;integer&quot;: verifying non-authoritative meta tag‌
Fetching https://k8s.io/utils?go-get=1
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;utils?go-get=1 (status code 200)‌
k8s.io&#x2F;utils (download)‌
Fetching https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;tools&#x2F;cache?go-get=1‌
Parsing meta tags from https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;tools&#x2F;cache?go-get=1 (status code 200)‌
get &quot;k8s.io&#x2F;client-go&#x2F;tools&#x2F;cache&quot;: found meta tag get.metaImport{Prefix:&quot;k8s.io&#x2F;client-go&quot;, VCS:&quot;git&quot;, RepoRoot:&quot;https:&#x2F;&#x2F;github.com&#x2F;kubernetes&#x2F;client-go&quot;} at https:&#x2F;&#x2F;k8s.io&#x2F;client-go&#x2F;tools&#x2F;cache?go-get=1‌
get "k8s.io/client-go/tools/cache": verifying non-authoritative meta tag
Removing intermediate container e388b0cd6310‌
 ---> 0db99f7f70f0
Step 5/10 : RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
 ---> Running in 3b7dc1880b91
# github.com&#x2F;sabbour&#x2F;phippy‌
./main.go:79:50: client.Core undefined (type *kubernetes.Clientset has no field or method Core)
The command &#x27;&#x2F;bin&#x2F;sh -c CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .&#x27; returned a non-zero code: 2‌
##[error]Bash exited with code '2'.
##[section]Finishing: docker build
```

**Comment by mathieu-benoit** [comment]
Thanks for your feedback @schdief, the associated fix #25 has been merged into master.

**Comment by schdief** [comment]
thanks works! :+1:
